README.md

```diff
@@ -118,3 +118,46 @@ func main() {
   req2.Header.Set("Authorization", cred.String())
 }
 ```
+
+## Custom Nonce Counter
+
+If the service does not provide unique nonces for multiple clients and you must not duplicate nonce counts.

```

Comment:
Seems like it would be easier to just get a new challenge for every request. Something like a `NoCache` flag.

Reply:
[Akamai Media Services for Live](https://learn.akamai.com/en-us/webhelp/media-services-live/media-services-live-encoder-compatibility-testing-and-qualification-guide-v4.0/GUID-8CD20D40-9629-41A4-B7F8-D6EA408BC417.html) sends all clients the same challenge (identical Opaque + Nonce values), rotating each hour. 🙃

Reply:
dang, ok.